build-tools/packages/build-cli/src/base.ts

```diff
@@ -49,4 +50,39 @@ export abstract class BaseCommand extends Command {
 
         return this._context;
     }
+
+    logWarn(message?: string, preMessage = "WARNING", ...args: unknown[]): void {
```

Comment:
(Not for this PR) I wonder if we should add a console utilities package or something. I could see utilities like this being useful for docs and test utils and such.

Reply:
Yup, I've been thinking the same thing. I'll add it once there's enough stuff to give it a clear "identity" (I don't want it to be a "whatever we needed to share" lib :smile:)